dotnet/src/Connectors/Connectors.Memory.SqlServer.Classic/SqlServerConfig.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Connectors.SqlServer.Classic;
+
+/// <summary>
+/// Configuration for the SQL Server memory store.
+/// </summary>
+public class SqlServerConfig
+{
+    /// <summary>
+    /// The default SQL Server memories table name.
+    /// </summary>
+    internal const string DefaultMemoryTableName = "SKMemories";
+
+    /// <summary>
+    /// The default SQL Server embeddings table name.
+    /// </summary>
+    internal const string DefaultEmbeddingsTableName = "SKEmbeddings";
```

Comment:
We should update these names and descriptions to match the new behavior.
Consider MemoryTablePrefix or BaseMemoryTableName.
It'll be good to explain in the summary how these names would be used as prefixes for the tables when a collection is created.